File: apps/institution-api/Models/VwCurrentInstitutionWebsite.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitutionWebsite
    {
        public int InstitutionWebsiteId { get; set; }
        public int InstitutionId { get; set; }
        public string InstitutionType { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionCode { get; set; }
        public string WebsiteUsage { get; set; }
        public int? ListSequence { get; set; }
        public string WebsiteUrl { get; set; }
        public Guid? WebsiteGuid { get; set; }
    }
}

File: libs/common/models/vw-current-institution-website.ts



export class VwCurrentInstitutionWebsite {
    public institutionWebsiteId: number = 0;
    public institutionId: number = 0;
    public institutionType: string = '';
    public institutionName: string = '';
    public institutionCode: string = '';
    public websiteUsage: string = '';
    public listSequence: number;
    public websiteUrl: string = '';
    public websiteGuid: string;

    constructor(init? : Partial<VwCurrentInstitutionWebsite>) {
        
        Object.assign(this, init);
    }
}

